client/my-sites/site-settings/comment-display-settings/index.jsx

```diff
@@ -0,0 +1,85 @@
+/**
+* External dependencies
+*/
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+import FormSelect from 'components/forms/form-select';
+import FormTextInput from 'components/forms/form-text-input';
+import JetpackModuleToggle from '../jetpack-module-toggle';
+import { isJetpackModuleActive } from 'state/selectors';
+import { getSelectedSiteId } from 'state/ui/selectors';
+
+class CommentDisplaySettings extends Component {
+	shouldEnableSettings() {
+		const { isCommentsModuleActive, submittingForm } = this.props;
+		return !! submittingForm || ! isCommentsModuleActive;
+	}
+
+	render() {
+		const {
+			fields,
+			onChangeField,
+			selectedSiteId,
+			submittingForm,
+			translate,
+		} = this.props;
+
+		return (
+			<FormFieldset className="comment-display-settings">
+				<JetpackModuleToggle
```

Comment:
As @rickybanister earlier said:

> Since the toggle alignment is just a couple of lines of css, could we just throw that commit in before we merge this?

I suggest we align it to the top before merging, it's a minor change that wouldn't hurt anyone.